great_expectations/core/util.py

```diff
@@ -140,6 +148,13 @@ def convert_to_json_serializable(data):
     elif isinstance(data, pd.DataFrame):
         return convert_to_json_serializable(data.to_dict(orient="records"))
 
+    elif pyspark and isinstance(data, pyspark.sql.DataFrame):
```

Comment:
Why not just `from pyspark.sql import DataFrame` and then not check for `pyspark` in line 151 and just say `isinstance(data, DataFrame)`?..

Reply:
Oh nevermind -- we use both Pandas and Spark here...  OK; maybe leave this as is just update the error message to not talk about `pyspark.sql` unless this was on purpose.  Thanks.

Reply:
done!